src/index.html

```diff
@@ -1,13 +1,468 @@
+        <article class="about-us__article" data-aos="fade-up">
+          <div class="about-us__content about-us__content-1">
+            <h2 class="about-us__title about-us__title-1">DESIGNED FOR THE FUTURE</h2>
+
+            <img
+              src="./images/sketch.png"
+              alt="sketches image"
+              class="about-us__img about-us__img-1"
+            >
+
+            <div class="about-us__text-content">
+              <p class="about-us__description">
+                In 2014, a group of geeky industrial designers, engineering veterans and acoustic experts formed
+                crazybaby.
+                This is a bunch of passionate people who are crazy enough to think they can challenge the industry with
+                disruptive audio products.
+              </p>
+              <a href="#" class="about-us__see-more">See more about us</a>
+
+            </div>
+          </div>
+
+        </article>
+
+        <article data-aos="fade-up" class="about-us__article">
+          <div class="about-us__content about-us__content-2">
+            <h2 class="about-us__title about-us__title-2">Luna Eye</h2>
+
+            <img
+              src="./images/pinkLuna.png"
+              alt="luna robot speaker image"
+              class="about-us__img about-us__img-2"
+            >
+            <div class="about-us__text-content">
+
+              <p class="about-us__description">
+                Luna Eye is to the essence of this innovative light and audio system. It comprises an independent
+                tweeter
+                speaker, a light guiding component, a dome and a ring ornament.
+              </p>
+              <a href="#" class="about-us__see-more">See more about us</a>
+            </div>
+          </div>
+        </article>
+      </section>
+
+      <div class="container">
+        <div class="main__content">
+          <section data-aos="fade-up" class="testimonials">
+            <p class="testimonials__comment">
+              “It really took me by surprise honestly to have such full beautiful sound that coming out of this small
+              compact device. And with the brush aluminum surface, it feels so familiar. Like my iPhone.”
+            </p>
+            <img
+              src="./images/avatar.png"
+              alt="avatar of Garret Martin"
+              class="testimonials__img"
+            >
+            <h5 class="testimonials__name">Garrett Martin</h5>
+            <p class="testimonials__profession">Creative Director</p>
+          </section>
+        </div>
+      </div>
+
+      <section
+        data-aos="fade-up"
+        class="features"
+        id="features"
+      >
+        <div class="container">
+
+          <h2 class="features__section-title">FEATURES</h2>
+
+          <div class="features__content">
+            <div class="features__cards">
+              <div class="features__card">
+                <h3 class="features__title">Connectivity</h3>
```

Comment:
What is the difference between `features__section-title` and `features__title`?
be better use `features__card-title`